src/test/java/ru/stepev/test/training/at/hw2/model/Page.java

```diff
@@ -0,0 +1,70 @@
+package ru.stepev.test.training.at.hw2.model;
+
+import java.util.Arrays;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class Page {
```

Comment:
As I could see all variables in this class looks mostly as a constants. 
I assume that in this case all fields could be declared as `public static final` 
Also class could be marked as a `final` with `private` constructor